src/sdl2/video.rs

```diff
@@ -1533,6 +1553,25 @@ impl Window {
         Ok(())
     }
 
+    #[doc(alias = "SDL_SetWindowShape")]
+    pub fn set_window_shape<S: AsRef<SurfaceRef>>(&mut self, shape: S) -> bool {
+        let mode = sys::WindowShapeMode::ShapeModeBinarizeAlpha;
+        let parameters = sys::SDL_WindowShapeParams{ binarizationCutoff: 0xFF};
+        let mut shape_mode = sys::SDL_WindowShapeMode {
+            mode,
+            parameters,
+        };
+        let result = unsafe {
+            sys::SDL_SetWindowShape(
+                self.context.raw,
+                shape.as_ref().raw(),
+                &mut shape_mode,
+            )
+        };
+        result == 0
```

Comment:
Could you please convert this bool into a `Result<(), _>`? I think in this case it will be `Result<(), u32>` or `Result<(), String>` according to the docs, but either way being able to see what kind of error is nicer than just false.

Reply:
Hi @Cobrand 
I fixed this to return Result<(), i32> (as the error code is i32 for SDL_SetWindowShape)